code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -413,15 +415,22 @@
 				P.name = trim("[name] bottle")
 				adjust_item_drop_location(P)
 				reagents.trans_to(P, vol_each, transfered_by = usr)
-		//SKYRAT EDIT ADDTION HYPOVIALS START
+		//SKYRAT EDIT ADDTION START
 		if(item_type == "vial")
 			var/obj/item/reagent_containers/glass/vial/small/P
 			for(var/i = 0; i < amount; i++)
 				P = new/obj/item/reagent_containers/glass/vial/small(drop_location())
 				P.name = trim("[name] vial")
 				adjust_item_drop_location(P)
 				reagents.trans_to(P, vol_each, transfered_by = usr)
-		//SKYRAT EDIT ADDTION HYPOVIALS END
+		if(item_type == "smartdart")
+			var/obj/item/reagent_containers/syringe/smartdart/P
+			for(var/i = 0; i < amount; i++)
```

Comment:
use `for(var/i in 1 to num)` (replace `num` with the amount of iterations that you want)